### PR TITLE
Add more retries for entreprise test case

### PR DESCRIPTION
**Pull request by anouarchattouna**
For more details see #208 and #209

### CODE REVIEW

File: test/consul_helpers.go [hunk 1]
```diff
@@ -196,8 +196,7 @@ func checkEnterpriseInstall(t *testing.T, asgNameOutputVar string, terratestOpti
 		SshUserName: sshUser,
 		SshKeyPair:  keyPair.KeyPair,
 	}
-
-	maxRetries := 10
+	maxRetries := 20
```
Comment (ina-stoyanova) [inline test/consul_helpers.go:199]: just curious - is there any derivation behind this value? I must be honest though - I'm lacking context, so excuse me if the question had been discussed before!

**Review by ina-stoyanova — APPROVED** [review]

File: test/consul_helpers.go (hunk 1 above)
Comment (anouarchattouna) [reply test/consul_helpers.go:199]: Actually, the idea is to add more retries for the `Check Enterprise Install` which is failing on `circleci` like you can see in this [comment](https://github.com/hashicorp/terraform-aws-consul/pull/208#issuecomment-784114002).
BTW: we are trying to find a way to fix #209

**Review by brikis98 — APPROVED** [review]
Let's give it a try! I'll kick off tests shortly.

**Comment by brikis98** [comment]
Tests passed. Re-running to see if they are more stable now, as this was an intermittent issue.

**Comment by brikis98** [comment]
Passed again! Merging now, thanks!

**Pull request merged**